repoindex.md

```diff
@@ -261,6 +261,34 @@ organization in GitHub under
 		</tr>
 	</table>
 </fieldset>
+<fieldset class="repo">
+	<legend>
+		pmem-related workloads
+	</legend>
+	<p>
+	These repos contain non-volatile memory related workloads.
+	Their concept is based on combined market ready solutions with
```

Comment:
They are meant to be production-ready solutions for PMem.

Reply:
updated